src/ActiveLogin.Authentication.BankId.AspNetCore/Events/BankIdCollectPendingEvent.cs

```diff
@@ -0,0 +1,22 @@
+using ActiveLogin.Authentication.BankId.Api.Models;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Events
+{
+    /// <summary>
+    /// Event for collect pending authentication order.
+    /// </summary>
+    public class BankIdCollectPendingEvent : BankIdEvent
+    {
+        public BankIdCollectPendingEvent(string orderRef, CollectHintCode hintCode)
```

Comment:
All the events should probably have internal constructors. So that only we can instantiate them.

Reply:
Changed all constructors to internal. Had to remove some tests to make this change. Perhaps we can come up with another way to test event listeners.